subprojects/language-java/src/integTest/groovy/org/gradle/api/tasks/JavaToolchainBuildOperationsIntegrationTest.groovy

```diff
@@ -130,33 +127,32 @@ class JavaToolchainBuildOperationsIntegrationTest extends AbstractIntegrationSpe
                 }
             }
 
-            tasks.register("$task", ToolchainTask) {
+            tasks.register("myToolchainTask", ToolchainTask) {
                 outputFile = layout.buildDirectory.file("output.txt")
                 launcher1 = javaToolchains.launcherFor {
                     languageVersion = JavaLanguageVersion.of(${jdkMetadata.languageVersion.majorVersion})
                 }
             }
-        """.stripIndent()
+        """
 
         when:
         runWithInstallation(jdkMetadata, task)
-        def events = eventsFor(taskPath)
+        def events = eventsFor(task)
         then:
-        executedAndNotSkipped(taskPath)
+        executedAndNotSkipped(task)
         assertToolchainUsages(events, jdkMetadata, "JavaLauncher")
 
         // TODO: should it work with up-to-date tasks?
```

Comment:
When this PR is in, we should extend the test to also get toolchain progress events under the `ResolveTaskMutationsBuildOperationType` operations. So this would work as well for up-to-date tasks.

Reply:
I think so as well. I'll wait until #21363 is merged and then try updating the tests for this